zoo-css-challenge/README.md

```diff
@@ -38,30 +34,26 @@ Change the background colour to #483C46, the section titles to #F4743B, and the
 
 Make the first paragraph have a larger font size.
 
-Again, the purple headings are hard to read on dark backgrounds, change them to a lighter colour.
```

Comment:
We just changed the headings in `Change the background colour to #483C46, the section titles to #F4743B` so the headings aren't purple anymore